core/common/assets/js/api/components/wordpress/commands-data/media.js

```diff
@@ -50,12 +55,16 @@ export class Media extends CommandData {
 
 		if ( this.file.size > parseInt( window._wpPluploadSettings.defaults.filters.max_file_size, 10 ) ) {
 			throw new Error( __( 'The file exceeds the maximum upload size for this site.', 'elementor' ) );
-		} else if (
+		}
+
+		if (
 			! window._wpPluploadSettings.defaults.filters.mime_types[ 0 ].extensions.split( ',' ).includes(
 				this.file.name.split( '.' ).pop()
-			)
+			) &&
+			! elementor.config.filesUpload.unfilteredFiles
 		) {
-			throw new Error( __( 'Sorry, this file type is not permitted for security reasons.', 'elementor' ) );
+			FilesUploadHandler.getUnfilteredFilesNotEnabledDialog( () => {} ).show();
+			return;
```

Comment:
On cancel from the SVG dialog, the panel still shows the dragged SVG

Reply:
Done.